### PR TITLE
Added docs for RDW sensor

### PR DESCRIPTION
**Pull request by eelcohn**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant# I will publish later and update this pull request with appropiate info.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by eelcohn** [comment]
More info on this sensor can be found on [this thread](https://community.home-assistant.io/t/custom-sensor-for-dutch-license-plate-checks-apk-check/94427).

**Comment by klaasnicolaas** [comment]
Wow nice that you build this component 😄

File: source/_components/rdw.markdown [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+The `rdw` platform allows you to integrate data from the [RDW](https://www.rdw.nl) (Netherlands Vehicle Authority). It will track the APK expiration date, the insurance state and notifications if the manufacturer of the car has issued a recall because of a serious safety problem. It is possible to add multiple cars with Dutch license plates to your Home Assistant configuration.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+sensor:
+  - platform: rdw
```
Comment (klaasnicolaas) [inline source/_components/rdw.markdown:26]: Put only required items in the first example.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/rdw.markdown (hunk 1 above)
Comment (eelcohn) [reply source/_components/rdw.markdown:26]: Is it allowed to provide two examples? A simple one with only the required items and a more complex one?

Comment (klaasnicolaas) [reply source/_components/rdw.markdown:26]: Yes ofcourse!

**Comment by eelcohn** [comment]
I still have some problems with moving the API calls to the pypi library. The original code works perfectly but after I moved the code with the API calls to the pypi library the sensors don't show in Home Assistant. The pypi library seems to be working correctly from the python command prompt.
I'll update this docs merge request when I've got the code working.

**Comment by klaasnicolaas** [comment]
If you need help with the code, please make a parent PR so that people can help you 😉

**Comment by frenck** [comment]
A month has passed, and still, there is no parent PR (at least, I did not find one).

I'm closing this PR. Feel free to re-open or create a new one when ready.

**Pull request closed without merging**